sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/SchedulerApiServer.java

```diff
@@ -33,6 +39,29 @@ public SchedulerApiServer(SchedulerConfig schedulerConfig, Collection<Object> re
                 new ResourceConfig(MultiPartFeature.class).registerInstances(new HashSet<>(resources)),
                 false /* don't start yet. wait for start() call below. */);
         this.startTimeout = schedulerConfig.getApiServerInitTimeout();
+
+        // Metrics
```

Comment:
statsd output to `STATSD_UDP_HOST`/`STATSD_UDP_PORT`?

Reply:
Addressed the rest of the comments.  Do you know if we use STATSD v2 or v3?

Reply:
Not sure what the difference is. They confusingly used the same name for the format and the default implementation of the server, maybe the v2 vs v3 thing is about that implementation? The format is just plain text over UDP. DC/OS metrics additionally supports the dogstatsd tags extension on the format: https://docs.datadoghq.com/guides/dogstatsd/#datagram-format